oranda-css/mdbook-theme/fonts/fonts.css

```diff
@@ -1,5 +1,6 @@
 /* fonts used by axo themes, not sure if this is the best way to fetch them */
 
+@import url("https://fonts.googleapis.com/css2?family=Fira+Sans:wght@400;700&display=swap");
```

Comment:
Oh right, I forgot to look into making this part more properly customizeable per-theme. One option that I think *might* work is to include these lines at the top of each theme variables file (the files where you declare `.oranda-dark`). Not something you need to fix, kinda just mumbling at myself here.

Reply:
@Gankra do you want to file an issue for this - i want to try to get this PR merged but i don't want to lose this idea as it sounds like you both prefer it

Reply:
https://github.com/axodotdev/oranda/issues/383